src/app/page/categoryInfo/course/course.component.ts

```diff
@@ -33,17 +33,17 @@ export class CourseComponent implements OnInit {
     this.courseService.getCourse(this.url).subscribe(
       courses => {
         this.courses = courses;
-        if (this.isAuthorized()) {
+        if (this.isAuthorized) {
           this.getCoursesIdOfTheUser();
         }
       });
     this.isAuthorized = this.logoutService.isAuthorized();
   }
 
-  onRatingClick = (course: CourseLink, event:IStarRatingOnClickEvent) => {
+  onRatingClick = (course: CourseLinkWithStatus, event:IStarRatingOnClickEvent) => {
```

Comment:
Because there is such a logic in that component.
CourseLink[] is used to initialize the courses list and then it is converted to CourseLinkWithStatus, where is the status field, which is true if the user is already subscribed to the course.

I'm not sure if it's okay but I used the same logic as in the deck component and the CourseLinkWithStatus class has been created before.